src/test/java/gov/usds/case_issues/authorization/CustomAccessDeniedHandlerTest.java

```diff
@@ -0,0 +1,29 @@
+package gov.usds.case_issues.authorization;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+
+import java.io.IOException;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.junit.Test;
+import org.springframework.http.HttpStatus;
+import org.springframework.security.access.AccessDeniedException;
+
+import gov.usds.case_issues.test_util.CaseIssueApiTestBase;
+
+public class CustomAccessDeniedHandlerTest extends CaseIssueApiTestBase {
```

Comment:
Does this actually need to run as a spring test, or is it fully functional with mocks? If so, can cut out the inheritance and run much faster.

Reply:
Removed. It is indeed not necessary